src/draw/lv_draw_label.c

```diff
@@ -101,7 +101,7 @@ LV_ATTRIBUTE_FAST_MEM void lv_draw_label_dsc_init(lv_draw_label_dsc_t * dsc)
     lv_memset_00(dsc, sizeof(lv_draw_label_dsc_t));
     dsc->opa = LV_OPA_COVER;
     dsc->color = lv_color_black();
-    dsc->font = LV_FONT_DEFAULT;
+    dsc->font = lv_font_get_normal();
```

Comment:
It seems risk with multiple displays if the displays' themes have different `font_normal`. This way always the current default display's font (set by `lv_disp_set_default(disp)`) will be used on all displays. 

Instead, the user can set a font for the screens in the theme. Now no font is set but we safely set `font_normal` (or `LV_FONT_DEFAULT` if it's `NULL`) in the screen style.